### ISSUE TITLE
Bug: createBdd returns Cucumber-Style Typing when using Playwright-Style with custom fixture in 6.5.0

### ISSUE THREAD

**Issue by ChristianHehlBM** [vitalets/playwright-bdd#163]
my fixtures.ts
```import { test as base } from "playwright-bdd";
import { MyFixture} from "./my-fixture";

export const test = base.extend<{ myFixture: MyFixture}>({
  myFixture: async ({ page }, use) => await use(new MyFixture(page)),
});
```

my steps.ts head:

```import { createBdd } from "playwright-bdd";
import { test } from "../support/fixtures";

export const { Given, When, Then } = createBdd(test);
```

I expect to have proper Typing for myFixture when calling `Given("...", async ({ myFixture }) => {})` but the function part of `Given` is typed as `CucumberStyleStepFn<never>`. 

When reverting to 6.4.0 the typing is correct.

**Comment by vitalets** [comment]
Can't reproduce, maybe some caching issue. 
Please provide a full demo using a template: https://github.com/vitalets/playwright-bdd-example 

![image](https://github.com/vitalets/playwright-bdd/assets/1473072/e0395862-bdf8-4a61-b962-cc66deeb52cb)

**Comment by ChristianHehlBM** [comment]
I was missing the `tsconfig.json` compared to the example you provided. I can reproduce it there by setting `strict: false`. This solves the problem for me.

I'm not that knowledgeable about TS to know, if this warrants further digging into the issue, or could be considered intended.

**Comment by grassynull33** [comment]
@vitalets I was able to confirm the issue that @ChristianHehlBM is having. I reverted to v6.4.0, and the types are now working correctly.

`page` has an `any` type on v6.5.1, and on v6.4.0, it has the correct type.

**Comment by vitalets** [comment]
@grassynull33 could you try to reproduce on https://github.com/vitalets/playwright-bdd-example?
Maybe it also depends on TS version.

**Comment by ChristianHehlBM** [comment]
here you go @vitalets https://github.com/vitalets/playwright-bdd-example/pull/18

**Comment by vitalets** [comment]
Thank you, @ChristianHehlBM ! Checking..

**Comment by vitalets** [comment]
Thanks @ChristianHehlBM @grassynull33! Fixed in **v6.5.2**.
Could you check on your side?

**Comment by grassynull33** [comment]
@vitalets thanks, confirmed working on the latest version you mentioned
